package.json

```diff
@@ -15,6 +15,7 @@
     "test:coverage": "jest --coverage",
     "test:routes": "jest -c jest.routes.config.js",
     "test:a11y": "jest -c jest.a11y.config.js --maxWorkers 15",
+    "test:e2e": "cucumber-js",
```

Comment:
Are these tests intended to replace the functional tests?

Reply:
The jenkins pipeline calls test:functional

Reply:
hmm, tbh i assumed so as in probate, functional test would call e2e however, i dont think theyre the same.